lib/utils/compose_ts.ts

```diff
@@ -189,8 +189,7 @@ export async function tarDirectory(
 		pack.entry(
 			{
 				name: toPosixPath(fileStats.relPath),
-				size: fileStats.stats.size,
-				mode: fileStats.stats.mode,
+				...fileStats.stats,
```

Comment:
This function applies when the `--nogitignore` option is used. When the option is not used, the `originalTarDirectory` function applies, so a similar change would presumably be also needed at:
https://github.com/balena-io/balena-cli/blob/v11.35.3/lib/utils/compose.js#L231

The plan is to de-duplicate the tarDirectory functions in the upcoming major version release, when the `--nogitignore` option would become the default behaviour (no-op).

Reply:
oh, done @pdcastro